Source/CesiumEditor/Private/CesiumGeoreferenceCustomization.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020-2021 CesiumGS, Inc. and Contributors
```

Comment:
I'll do a pass that corrects the copyrights in these files and add them as a commit.

```suggestion
// Copyright 2020-2023 CesiumGS, Inc. and Contributors
```